Content.Server/Disposal/Unit/EntitySystems/DisposableSystem.cs

```diff
@@ -112,8 +114,10 @@ public void ExitDisposals(EntityUid uid, DisposalHolderComponent? holder = null,
                 if (duc != null)
                     duc.Container.Insert(entity, EntityManager, xform, meta: meta);
                 else
+                {
                     _xformSystem.AttachToGridOrMap(entity, xform);
-
+                    _throwing.TryThrow(entity, _random.NextVector2(0.1f), 0.1f);
+                }
                 if (EntityManager.TryGetComponent(entity, out PhysicsComponent? physics))
```

Comment:
Also, throwing the entity should wake it up so you can remove this bit as it is now redundant.